### PR TITLE
[FIX] pos_mercado_pago: allow sending payment request

### PR DESCRIPTION
**Pull request by sbel-odoo**
Currently, when selecting the Mercago Pago payment method, the payment is directly shown as being cancelled, even though is wasn't even sent in the first place.

Steps to reproduce:
-------------------
* Go to the **Point of Sale** App
* Under **Configuration** select **Payment Mehods**
* Create a new payment method and configure it with Mercado Pago
* Under **Configuration** select **Settings**
* Add the payment method to the shop
* Open shop session
* Create an order and go to pay it
* Select the Mercado Pago payment method created
> Observation: Transaction cancelled

Why the fix:
------------
When adding selecting the payment method we would calll `send_payment_request` which would fail at multiple location because we were trying to retreive a property from `underfined`.

opw-4142677

### CODE REVIEW

File: addons/pos_mercado_pago/static/src/app/payment_mercado_pago.js [hunk 1]
```diff
@@ -6,40 +6,40 @@ import { AlertDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
...
         const order = this.pos.get_order();
-        const line = order.selected_paymentline;
+        const line = order.get_selected_paymentline();
         // Build informations for creating a payment intend on Mercado Pago.
         // Data in "external_reference" are send back with the webhook notification
         const infos = {
             amount: parseInt(line.amount * 100, 10),
             additional_info: {
-                external_reference: `${this.pos.config.current_session_id.id}_${line.payment_method.id}_${order.uid}`,
+                external_reference: `${this.pos.config.current_session_id.id}_${line.payment_method_id.id}_${order.pos_reference}`,
```
Comment (davidmonnom) [inline addons/pos_mercado_pago/static/src/app/payment_mercado_pago.js:15]: Can you use order UUID ?

```suggestion
                external_reference: `${this.pos.config.current_session_id.id}_${line.payment_method_id.id}_${order.uuid}`,
```

**Review by davidmonnom — CHANGES_REQUESTED** [review]
Just a little change otherwise LGTM

File: addons/pos_mercado_pago/static/src/app/payment_mercado_pago.js (hunk 1 above)
Comment (sbel-odoo) [reply addons/pos_mercado_pago/static/src/app/payment_mercado_pago.js:15]: I think I could... I read the PR which introduced this very recently https://github.com/odoo/odoo/pull/177300 and he specifically wanted to add `uid/pos_reference` as an identifier so I stayed consistent with this. But I guess the `uuid`could work as well.

I won't be able to test that part though. I don't have access to mercado in Belgium, I'm using credentials from MX. But anyway Yaro says that they have problems with Mercado Pago terminal in Belgium and this behavior cannot be tested.

What do you think?

Comment (davidmonnom) [reply addons/pos_mercado_pago/static/src/app/payment_mercado_pago.js:15]: Yes, you can't really test it, but in general in PoS, the unique identifier of an order is the UUID.

Comment (sbel-odoo) [reply addons/pos_mercado_pago/static/src/app/payment_mercado_pago.js:15]: I'll change that right away then :+1:

**Comment by davidmonnom** [comment]
@robodoo r+

**Pull request closed without merging**